test/crio_secrets.bats

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env bats
+
+load helpers
+
+function teardown() {
+	cleanup_test
+}
+
+IMAGE="redis:alpine"
+ROOT="$TESTDIR/crio"
+RUNROOT="$TESTDIR/crio-run"
+KPOD_OPTIONS="--root $ROOT --runroot $RUNROOT ${STORAGE_OPTS}"
+
+MOUNT_PATH_1="$TESTDIR/secrets"
+mkdir ${MOUNT_PATH_1}
+MOUNT_FILE_1="${MOUNT_PATH_1}/test.txt"
+touch ${MOUNT_FILE_1}
+echo "Testing secrets mounts!" > ${MOUNT_FILE_1}
+
+MOUNT_PATH_2="$TESTDIR/more_secrets"
+mkdir ${MOUNT_PATH_2}
+MOUNT_FILE_2="${MOUNT_PATH_2}/test_2.txt"
+touch ${MOUNT_FILE_2}
+echo "Testing secrets mounts again!" > ${MOUNT_FILE_2}
+
+echo "${MOUNT_PATH_1}:/container/path1" >> ${SECRETS_MOUNTS_FILE}
+echo "${MOUNT_PATH_2}:/container/path2" >> ${SECRETS_MOUNTS_FILE}
+
+@test "bind secrets mounts to container" {
+    start_crio
+    run crioctl pod run --config "$TESTDATA"/sandbox_config.json
+    echo "$output"
+    [ "$status" -eq 0 ]
+    pod_id="$output"
+    run crioctl image pull "$IMAGE"
+    [ "$status" -eq 0 ]
+    run crioctl ctr create --config "$TESTDATA"/container_redis.json --pod "$pod_id"
+    echo "$output"
+    [ "$status" -eq 0 ]
+    ctr_id="$output"
+    run bash -c "${KPOD_BINARY} ${KPOD_OPTIONS} ps -a --format json | python -m json.tool | grep -wE '(${MOUNT_PATH_1}|${MOUNT_PATH_2})'"
```

Comment:
use crioctl, not kpod

Reply:
also, what are you testing? use `crioctl execsync --id $ID mount | grep {for the secret mount you're injecting}`

Reply:
I think you would actually want to make sure that a file exists at the mount point in addition to seeing that the directory is mounted.

Reply:
Could you take care of this one?

Reply:
I have been trying to do
```
 run crioctl ctr create --config "$TESTDATA"/container_redis.json --pod "$pod_id"
    echo "$output"
    [ "$status" -eq 0 ]
    ctr_id="$output"
    run bash -c "crioctl ctr execsync --id $ctr_id | grep -wE '(${MOUNT_PATH_1}|${MOUNT_PATH_2})'"
    echo "$output"
    [ "$status" -eq 0 ]
```
But I keep getting a "container with ID ... not found" error. However it works with kpod ps, even though I use `bash -c` in both cases.
@runcom I also sent you an email explaining it a bit more in detail.

kpod ps displays the mounts of the container, so isn't that good enough for testing purposes?

Reply:
Doesn;t execsync require you to add a command?
```
run bash -c "crioctl ctr execsync --id $ctr_id mount
```

Reply:
@rhatdan yup, I have been trying it with the mount command but still keep getting the container not found error. When I run it without the bash -c, it runs fine.
From what I read about bash -c, it is basically using another shell so the test environment is not retained. Not sure why kpod ps works fine with bash -c though.

Reply:
Well you could hack this with

See if adding an 
export ctr_id=$output

Fixes your issue.

Reply:
@rhatdan nope that doesn't fix it. It is getting the value of `ctr_id` but does not find a container with such an id when ran using `bash -c`.
I also tried writing the output to a temp file and running grep on it for the mounts, but it seems like the data in the file is in the form of a byte array.
@nalind suggested some things, which I tried but never got the tests to pass :/
It is working and the tests pass with `kpod ps`, so is it really important to test it with `crioctl ctr execsync` instead? FWIW I see the mounts with `crioctl ctr execsync mount` when done manually.

Reply:
How about if you run it without the bash -c at all and then just grep'd the $output for the content?